pkg/spec/spec.go

```diff
@@ -31,14 +31,17 @@ type Function struct {
 
 // FunctionSpec contains func specification
 type FunctionSpec struct {
-	Handler  string             `json:"handler"`
-	Function string             `json:"function"`
-	Runtime  string             `json:"runtime"`
-	Type     string             `json:"type"`
-	Topic    string             `json:"topic"`
-	Schedule string             `json:"schedule"`
-	Deps     string             `json:"deps"`
-	Template v1.PodTemplateSpec `json:"template" protobuf:"bytes,3,opt,name=template"`
+	Handler     string             `json:"handler"`      // Function handler: "file.function"
+	Function    string             `json:"function"`     // Function file content or URL of the function
```

Comment:
I didn't see anything here that would support a URL - and mashing a URL or code text into the same textual value seems hazardous.

Reply:
Yes, in this PR it is not supported to use a URL, that's for the Minio one.